autoPyTorch/api/base_task.py

```diff
@@ -253,13 +260,15 @@ def __init__(
                 raise ValueError("Expected search space updates to be of instance"
                                  " HyperparameterSearchSpaceUpdates got {}".format(type(self.search_space_updates)))
 
+        self.time_series_forecasting = task_type == 'time_series_forecasting'
```

Comment:
Based on [this](https://github.com/automl/Auto-PyTorch/pull/434/files#r895920906) comment, I think you can remove this flag. If you would like to keep it can I suggest you move the logic to the `TimeSeriesForecastingTask`? You can initialise it in `BaseTask` as False. Moreover, as we dont want to have undefined constant strings in the code, could you check this similar to 
```
if STRING_TO_TASK_TYPE[task_type] in FORECASTING_TASKS
```